docs/en/integrations/language-clients/java/jdbc-v2.md

```diff
@@ -0,0 +1,141 @@
+- Enum - can be treated as both string and integer
+- UInt64 - mapped to `long` (in client-v1) 
+:::
+
+## Creating Connection
+
+```java
+String url = "jdbc:ch://my-server:8123/system"; // use http protocol
+
+Properties properties = new Properties();
+
+DataSource dataSource = new DataSource(url, properties);
+try (Connection conn = dataSource.getConnection("default", "password");
+    Statement stmt = conn.createStatement()) {
+}
+```
+
+## Simple Statement
+
+```java showLineNumbers
+
+try (Connection conn = dataSource.getConnection(...);
+    Statement stmt = conn.createStatement()) {
+    ResultSet rs = stmt.executeQuery("select * from numbers(50000)");
+    while(rs.next()) {
+        // ...
+    }
+}
+```
+
+## Insert
+
+```java showLineNumbers
+try (PreparedStatement ps = conn.prepareStatement("INSERT INTO mytable VALUES (?, ?)")) {
+    ps.setString(1, "test"); // id
+    ps.setObject(2, LocalDateTime.now()); // timestamp
+    ps.addBatch();
+    ...
+    ps.executeBatch(); // stream everything on-hand into ClickHouse
+}
+```
+## Migrating from 0.7.x
+In general we tried to make the driver more strictly follow the JDBC specification, so there are some changes that may affect you:
+
+| Old Feature                     | Notes                                                                                                                                                                                                     |
+|---------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Transaction Support             | Early versions of the driver only **simulated** transaction support, which could have unexpected results.                                                                                                 |
+| Response Column Renaming        | ResultSets were read/write - for efficiency sake they're now read-only                                                                                                                                    |
+| Multi-Statement SQL             | Statements would split multi-statements and execute, now it strictly follows 1:1                                                                                                                          |
+| Named Parameters                |                                                                                                                                                                                                           |
+| Stream-based PreparedStatements | Early version of the driver allowed for non-jdbc usage of PreparedStatements - if you desire such options, we recommend looking to [Client-V2](/docs/en/integrations/language-clients/java/client-v2.md). |
+
+
+
+## More Information
```

Comment:
the following parts exist in JDBC v1, but not v2. Is that intentionally? 
https://clickhouse-docs-el5b4zr1v-clickhouse.vercel.app/docs/en/integrations/java/jdbc-v1#configuring-http-library

https://clickhouse-docs-el5b4zr1v-clickhouse.vercel.app/docs/en/integrations/java/jdbc-v1#connect-to-clickhouse-with-ssl

https://clickhouse-docs-el5b4zr1v-clickhouse.vercel.app/docs/en/integrations/java/jdbc-v1#resolving-jdbc-timeout-on-large-inserts